### PR TITLE
[API] fix failing test by enabling api

### PR DESCRIPTION
**Pull request by SirDomin**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/test/src/Tests/DisablingApiTest.php [hunk 1]
```diff
@@ -29,9 +29,11 @@ public function setUp(): void
     /** @test */
     public function it_gets_collection_if_api_is_enabled(): void
     {
+        $this->enableApi();
+
         static::createClient()->request(
             'GET',
-            'api/v2/admin/orders',
+            'api/v2/admin/countries',
```
Comment (Zales0123) [inline src/Sylius/Bundle/ApiBundle/test/src/Tests/DisablingApiTest.php:36]: Why it needs to be changed?

Comment (SirDomin) [reply src/Sylius/Bundle/ApiBundle/test/src/Tests/DisablingApiTest.php:36]: it was a quick solution for the problem, it seems like /orders endpoint was changed between 1.9 and master as it was working fine with 1.9 version.

Comment (Zales0123) [reply src/Sylius/Bundle/ApiBundle/test/src/Tests/DisablingApiTest.php:36]: How does it then works in [this test](https://github.com/SirDomin/Sylius/blob/fix-disabled-api-master/src/Sylius/Bundle/ApiBundle/test/src/Tests/DisablingApiTest.php#L61)? 😱

**Comment by GSadee** [comment]
Closing as it has been already fixed in other PR

**Pull request closed without merging**